web/cobrands/sass/_layout.scss

```diff
@@ -671,11 +671,13 @@ body.authpage {
 
   &:after {
     content: "";
-    display: inline-block;
-    vertical-align: middle;
```

Comment:
Just curious as to why top 50% -1/2 height is better than this (I'm sure it is, just for my own benefit).